.devops/templates/runpublishvrscreenshot.yml

```diff
@@ -67,7 +67,7 @@ steps:
 
```

Comment:
## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentuiv9 Visual Regression Report](https://aka.ms/visual-changes-review?organization=uifabric&project=fabricpublic&prId=33007&commitId=12947c928d4aace3adc1505d2d89356b92b2169a&env=prod&clientType=FLUENTUI)


  <details><summary><b>Avatar Converged</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar Converged.badgeMask.chromium.png|8| Changed|

</details>


  <details><summary><b>Drawer</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Drawer.overlay drawer full.chromium.png|1154| Changed|
|Drawer.Full Overlay High Contrast.chromium.png|4944| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>